Importing direct LUNs from RHV (#416)
kubev2v/forklift-documentation: documentation/modules/rhv-prerequisites.adoc

* You must have the {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate.
+
You can obtain the {manager}  CA certificate by navigating to https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA in a browser.

> * If you are migrating a direct LUN disk, ensure that the nodes in the {virt} destination cluster, especially the node that is going to run the VM, has access to the backend storage.

Reviewer: ```suggestion
* If you are migrating a virtual machine with a direct LUN disk, ensure that nodes in the {virt} destination cluster that the VM is expected to run on, have access to the backend storage.
```

* You must have the {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate.
+
You can obtain the {manager}  CA certificate by navigating to https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA in a browser.

> * If you are migrating a virtual machine with a direct LUN disk, ensure that the nodes in the {virt} destination cluster that the VM is expected to run on can access the backend storage.

Reviewer 2: ```suggestion
* If you are migrating a virtual machine with a direct LUN disk, ensure that the nodes in the {virt} destination cluster that the VM is expected to run on, can access the back end storage.
```
